readme.md

```diff
@@ -181,7 +181,7 @@ Let us do some maths first before we write any Python code:
 - 3 - 2 = 1
 - 3 \* 2 = 6
 - 3 / 2 = 1.5
-- 3 ^ 2 = 3 x 3 = 9
+- 3 \*\* 2 = 3 x 3 = 9
```

Comment:
why backward slash?
3 ** 2 = 9 
where 3*3 = 9

Reply:
The backslash escapes the asterisk. When rendering a preview of the .md file,  ** will change the formatting of the line